attributes/default.rb

```diff
@@ -225,7 +225,11 @@
   { type: 'host', db: 'all', user: 'all', addr: '::1/128', method: 'md5' }
 ]
 
-default['postgresql']['password'] = {}
+default['postgresql']['password'] = nil
```

Comment:
This needs to go back to a {}. I meant the commented out attribute below (default['postgresql']['pg_gem']['version']) should be set to nil